### PR TITLE
Install CA root keys as package

### PR DESCRIPTION
**Pull request by originalsouth**
### Changes
Install CA root keys as package instead of digging it.

### Issue link
N/A

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by dekkers — COMMENTED** [review]
We should also remove the `-k root.key` from the drill command here https://github.com/minvws/nl-kat-coordination/blob/631c44af3c05d6ab8b80e8fa774e2a44e269e475/boefjes/boefjes/plugins/kat_dnssec/main.py#L20

File: boefjes/boefjes/plugins/kat_dnssec/main.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def run(boefje_meta: dict):
         )
 
     remote_ns = os.getenv("REMOTE_NS", "1.1.1.1")
-    cmd = ["/usr/bin/drill", "-S", "-k", "root.key", str(domain), "@" + remote_ns]
+    cmd = ["/usr/bin/drill", "-S", "-k", "/usr/share/dns/root.key", str(domain), "@" + remote_ns]
```
Comment (dekkers) [inline boefjes/boefjes/plugins/kat_dnssec/main.py:20]: This is already the default, so we can simplify it with

```suggestion
    cmd = ["/usr/bin/drill", "-S", str(domain), "@" + remote_ns]
```

**Review by Donnype — APPROVED** [review]
Cool :+1:, also felt there must be a better way when I was moving the code

**Comment by originalsouth** [comment]
> Cool 👍, also felt there must be a better way when I was moving the code

Courtesy of @dekkers :smile:

**Review by dekkers — APPROVED** [review]

**Pull request merged**